crates/db_schema/src/source/mod_log/moderator.rs

```diff
@@ -210,29 +213,28 @@ pub struct ModBan {
 }
 
 #[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
-#[cfg_attr(feature = "full", diesel(table_name = mod_hide_community))]
-pub struct ModHideCommunityForm {
+#[cfg_attr(feature = "full", diesel(table_name = mod_change_community_visibility))]
+pub struct ModChangeCommunityVisibilityForm {
   pub community_id: CommunityId,
   pub mod_person_id: PersonId,
-  pub hidden: Option<bool>,
   pub reason: Option<String>,
+  pub new_visibility: CommunityVisibility,
```

Comment:
I'd prefer this just be called `visibility`.

Reply:
Changed